packages/wrangler/src/metrics/send-event.ts

```diff
@@ -0,0 +1,108 @@
+import { logger } from "../logger";
+import { getMetricsDispatcher } from "./metrics-dispatcher";
+import type { Config } from "../config";
+import type { Properties } from "./metrics-dispatcher";
+
+export type EventNames = DashEventNames | WranglerEventNames;
+
+/** These are related event names currently in use in the dashboard. */
+type DashEventNames =
+	| "add cron trigger"
+	| "add workers route"
+	| "begin zone activation flow"
+	| "begin log stream"
+	| "change zone setting"
+	| "Change zone setup confirmation"
+	| "change zone status"
+	| "Change zone status confirmation"
```

Comment:
these are capitalised, intentional?

Reply:
All these `DashEventNames` are current names in the CF dashboard app. As such they are not all following the correct format. It turns out we are not using any of them, so I am just going to remove this type.